src/main/java/br/unb/cic/analysis/svfa/confluence/SVFAConfluenceAnalysis.java

```diff
@@ -34,13 +33,13 @@ public Set<ConfluenceConflict> getConfluentConflicts() {
      * the confluentFlows attribute with the results
      */
     public void execute() {
-        SVFAAnalysis sourceBaseAnalysis = sourceBaseAnalysis(interprocedural);
-        sourceBaseAnalysis.buildSparseValueFlowGraph();
-        Set<List<LambdaNode>> sourceBasePaths = sourceBaseAnalysis.findSourceSinkPaths();
+        DFPAnalysisSemanticConflicts sourceBaseAnalysis = sourceBaseAnalysis(interprocedural);
+        sourceBaseAnalysis.buildDFP();
```

Comment:
@galilasmb @pauloborba essa mudança para confluence usar DFP é permanente? Se não for,  acho que não faz sentido realizarmos essa alteração.

Reply:
é permanente sim, mas teria sido mais adequado criar uma nova classe, ou renomear essa

Reply:
Vou renomear a classe.